.ci/openshift-ci/build_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script is evoked within an OpenShift Build to product the binary image,
+# which will contain the Kata Containers installation into a given destination
+# directory.
+#
+
+set -e
+
+export PATH="${GOPATH}/bin:$PATH"
+
+cidir="$(dirname "$0")/.."
+source /etc/os-release
+
+source "${cidir}/lib.sh"
+
+[ -z "$1" ] && die "Usage: $0 path/to/install/dir"
+export DESTDIR="$1"
+info "Build and install Kata Containers at ${DESTDIR}"
+
+[ "$(id -u)" -ne 0 ] && die "$0 must be executed by privileged user"
+
+# Let the scripts know it is in OpenShift CI context.
+export OPENSHIFT_CI="true"
+
+# This script is evoked within a variant of CentOS container image in the
+# OpenShift Build process. So it was implemented for running in CentOS.
+[ "$ID" != "centos" ] && die "Expect the build root to be CentOS"
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
+"${cidir}/setup_env_centos.sh" default
+
+# The build root container has already golang installed, let's remove it
+# so that it will use the version required by kata.
+yum remove -y golang
```

Comment:
do we need `sudo`?

Reply:
Hi @GabyCT , thanks for your approval!

It doesn't need `sudo` because the script is executed with admin privileges. I've also added a check on the script that if not admin then exit with error (as suggested by @jodh-intel )